src/renderer/components/+helm-charts/get-char-details.injectable.ts

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) OpenLens Authors. All rights reserved.
+ * Licensed under MIT License. See LICENSE in root directory for more information.
+ */
+import { getInjectable } from "@ogre-tools/injectable";
+import type { GetChartDetailsOptions, IHelmChartDetails } from "../../../common/k8s-api/endpoints/helm-charts.api";
+import { getChartDetails } from "../../../common/k8s-api/endpoints/helm-charts.api";
+
+export type GetChartDetails = (repo: string, name: string, opts?: GetChartDetailsOptions) => Promise<IHelmChartDetails>;
+
+const getChartDetailsInjectable = getInjectable({
+  id: "get-chart-details",
+  instantiate: (): GetChartDetails => getChartDetails,
+  causesSideEffects: true,
+});
+
+export default getChartDetailsInjectable;
```

Comment:
@jansav Do we need to create an injectable for such cases where there're no injectables inside of it? Why not just use `getChartDetails` function directly?

Reply:
This is so that I can override it in tests. In the future it would be better to make this sort of route fully injectable but that would take too long.

Reply:
@Nokel81 approach seems right. In the future we might want to override something else which `getChartDetails` gets as dependency. But for now this seems fine.